Cargo.toml

```diff
@@ -12,31 +12,31 @@ keywords = ["github", "webhook"]
 edition = "2018"
 
 [build-dependencies]
-cfgrammar = "0.9"
-lrlex = "0.9"
-lrpar = "0.9"
-rerun_except = "0.1"
+cfgrammar = { path="../grmtools/cfgrammar"}
+lrlex = { path="../grmtools/lrlex"}
+lrpar = { path="../grmtools/lrpar"}
+rerun_except = "1.0"
 
 [dependencies]
-cfgrammar = "0.9"
-crypto-mac = "0.10"
+cfgrammar = { path="../grmtools/cfgrammar"}
+crypto-mac = "0.11"
 getopts = "0.2"
 hex = "0.4"
-hmac = "0.10"
+hmac = "0.11"
 hyper = { version = "0.14", features = ["http1", "server", "tcp"] }
 json = "0.12"
 libc = "0.2"
-lrlex = "0.9"
-lrpar = "0.9"
-nix = "0.19"
+lrlex = { path="../grmtools/lrlex"}
```

Comment:
do these need to be build deps if they are already normal deps?

Reply:
Yes, grmtools does things at compile-time and run-time.

Reply:
k